files/jquery.loadingoverlay/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Gaspare Sganga"
+github = "https://github.com/gasparesganga/jquery-loading-overlay"
+homepage = "http://gasparesganga.com/labs/jquery-loading-overlay/"
```

Comment:
Yep, that's the official documentation page for this jquery plugin. Thanks for reviewing my pull request!